src/data/markdown/translated-guides/en/01 Getting started/04 Results Output.md

```diff
@@ -111,16 +111,16 @@ If the aggregated [end-of-test summary](/results-visualization/end-of-test-summa
 
 The available built-in outputs currently are:
 
-| Plugin                                                        | Usage                   |
+| Output                                                        | Usage                   |
```

Comment:
I think needs to be reworked ... as we have neither plugins (at all) nor outputs for Amazon Cloudwatch, New relic and soon datadog and kafka. 

But it is probably still worth it to document how to use those results "outputs"

Reply:
Agreed, especially because we mention "built-in outputs" above, which these aren't.

I think we should just list the real built-in outputs here, and then on each page where that output could be used for several services, like with statsd, mention which services are supported and examples for each. We don't need separate pages for services that use the same output.

Can you create an issue for this so that we get to it soon? Or do it in this PR if you have time :)